### PR TITLE
Add Appearances page and added Appearance button in navbar to redirect to Appearances Page

### PR DESCRIPTION
**Pull request by Tamanna225**
Issue: #24

**Features Added:** Created a new Page of Appearances of Tony Stark(Added Appearances.html and Appearances.css)
**Changes Made:** Added the Appearances Button navbar to redirect to Appearances page
**Responsiveness:** Responsive for All the screens
All the other features same as index.html

**Mobile View:**

Screenshot 2024-10-03 191443

**Laptop View**

Screenshot 2024-10-03 191512

If any further changes required Let me know

### CODE REVIEW

File: Appearances.css [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+  color: white;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  max-width: 50%;
+  margin: auto;
+  height: 80%;
+}
+
+  
+/* Adjusted Logo */
+.logo img {
```
Comment (aslams2020) [inline Appearances.css:195]: Please see that the main logo of the website is reduced in size.

Original : 
image

After your changes - 
image

**Review by aslams2020 — CHANGES_REQUESTED** [review]
The changes In Summary you have to make - 

1. See the Main-logo in Navbar. **Get it's size back to normal.**
2. Replace the **"Merchandise"** Link with **"Appearances" link**.. Instead of Newly creating a link in navbar.
3. In Appearances Page : **Change the Colour Of the Main Heading**. It is not following the web theme.
image

**Comment by Tamanna225** [comment]
Made the required changes

**Review by aslams2020 — APPROVED** [review]
Approved

**Comment by aslams2020** [comment]
@Tamanna225 Hi, 
I got My account back today. On next leaderboard update you'll be able to see your points. 
Sorry for the inconvenience. Keep contributing.🚀

**Pull request merged**